src/locales/ja.yaml

```
@@ -196,26 +225,35 @@ app:
       host: ホスト
       layer: レイヤー
       layout: レイアウト
+      ldap: LDAP
       longest_job: 長い印刷
       low: 低い
       m117: M117
+      moonraker: Moonraker
       name: 名前
       new_password: 新しいパスワード
       no_notifications: 通知なし
       'off': オフ
       'on': オン
+      partial_of_total: '  %{total}の%{partial} '
```

I don't think this translation should start with whitespaces
```suggestion
      partial_of_total: '%{total}の%{partial} '
```